### DISCUSSION TITLE
notes/vladimir-de-turckheim/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#11] [Announcements]
# notes/vladimir-de-turckheim/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/vladimir-de-turckheim/

**Comment by vim-usds** [comment]
slides here - https://docs.google.com/presentation/d/1NVyRgitg-2CyN3BuoZZF6F-Dw8PuDybFjkFICAyYPHo/edit#slide=id.p1
